src/components/MyActivities/MyActivitiesStyle.scss

```diff
@@ -220,3 +220,43 @@
   height: 40px;
   padding-right: 0.5rem;
 }
+
+.overlay-help {
+  position: absolute;
+  width: 70px;
+  height: 70px;
+  box-shadow: 0 0 0 9999px rgba(28, 35, 65, 0.95);
```

Comment:
Maybe you could add an animation for this shadow. And also a small radius for this shadow is better :))